### DISCUSSION TITLE
is there a make uninstall or equivalent?

### DISCUSSION THREAD

**Discussion by BobManMasher** [gerbera/gerbera#1224] [Q&A] [answered]
Having built the latest Gerbera build, I went to test it using make install which copied all the necessarily files to all the correct places but didn't set up all the directories etc or configure correctly for init.d which I need on my Pogo box. The service and files were also copied into the wrong areas. I wanted to remove the files and tried make uninstall and got the following message:

`make: *** No rule to make target 'uninstall'.  Stop.
`
What is the best / correct way to remove the files after a make install?

**Comment by whyman** [comment, marked as answer]
It looks like CMake doesnt generate one automatically, which is surprising.

The install section is pretty small though, so you shouldnt have much to remove manually:

https://github.com/gerbera/gerbera/blob/11b9273b75973aa05c753cfdeaba4227f2c5f207/CMakeLists.txt#L555-L562

Will be under `/usr/local` by default.

**Reply by BobManMasher** [reply]
Thanks @whyman - Will have a look at that and manually remove the files.  One of my problems with the init.d is that the files need to be in /usr/bin and not /usr/local/bin so now I know the file locations, I will try to manually copy them into the right place.  Can you add an uninstall script in a future version and should I post a request?
